python/test/hail/expr/test_expr.py

```diff
@@ -1439,7 +1439,9 @@ def test_sorted(self):
         self.assertEqual(hl.eval_expr(hl.sorted([0, 1, 4, 3, 2], lambda x: x % 2, reverse=True)), [1, 3, 0, 4, 2])
 
         self.assertEqual(hl.eval_expr(hl.sorted([0, 1, 4, hl.null(tint), 3, 2], lambda x: x)), [0, 1, 2, 3, 4, None])
-        self.assertEqual(hl.eval_expr(hl.sorted([0, 1, 4, hl.null(tint), 3, 2], lambda x: x, reverse=True)), [None, 4, 3, 2, 1, 0])
+        # FIXME: this next line triggers a bug: None should be sorted last!
+        # self.assertEqual(hl.sorted([0, 1, 4, hl.null(tint), 3, 2], lambda x: x, reverse=True).collect()[0], [None, 4, 3, 2, 1, 0])
```

Comment:
I want to leave this. It's a bug which got revealed by this change, but is orthogonal. I'll make an issue when this goes in

Reply:
I guess I'm confused because the uncommented line does have None at the end and the expression to generate the result looks identical to me. Maybe I'm missing something. Or do you mean None should be at the front of the reversed list. I thought this was the intended behavior.

Reply:
Oh, sorry! This line succeeds but the test is wrong. None should be at the end whether reversed or not. This is the semantics of Interpret, but not emit.